integration-webhook/src/main/java/org/symphonyoss/integration/webhook/WebHookIntegration.java

```diff
@@ -183,13 +181,7 @@ private void authenticate(String integrationUser) {
     try {
       authenticationProxy.authenticate(integrationUser);
     } catch (RemoteApiException e) {
-      exceptionHandler.handleAuthenticationApiException(integrationUser, e);
-    } catch (ConnectivityException e) {
-      // Needed to rethrow this specific exception, otherwise it will endup hidden under the
-      // catch bellow.
-      throw e;
-    } catch (Exception e) {
-      exceptionHandler.handleAuthException(e);
+      throw new RetryLifecycleException("Unexpected error when authenticating", e);
```

Comment:
I believe you should catch Exception and throw a UnexpectedAuthException exception.

This way you can define a well-defined message and in the future we can define a solution.